server/controllers/tracing-controller.js

```diff
@@ -0,0 +1,120 @@
+const knex = require("./../db");
+const moment = require("moment");
+const checkManager = async (userId)=>{
+    const result = await knex
+    .select({ id: "id", role: "role" })
+    .from("user")
+    .where("id", userId)
+    .catch(() => {
+        throw { msg: `There was an error retrieving user info`, status: 400 };
```

Comment:
I think most controllers use the format {message: `error message` } when replying with errors.

Reply:
There are other instances of this in the file, but I won't comment on them.